src/fragmentarium/ui/fragment/WordDownloadButton.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react'
+import { wordExport } from './WordExport'
+import { Fragment } from 'fragmentarium/domain/fragment'
+import WordService from 'dictionary/application/WordService'
+import { Dropdown } from 'react-bootstrap'
+import { saveAs } from 'file-saver'
+import Spinner from 'common/Spinner'
+import { Packer } from 'docx'
+
+type Props = {
+  children: React.ReactNode
+  fragment: Fragment
+  wordService: WordService
+}
+
+export default function WordDownloadButton({
+  fragment,
+  wordService,
+  children,
+}: Props): JSX.Element {
+  const [isLoading, setIsLoading] = useState(false)
+
+  const handleClick = (event) => {
+    setIsLoading(true)
+    getWordBlob(fragment, wordService).then((blob) => {
+      saveAs(blob, `${fragment.number}.docx`)
+      setIsLoading(false)
+    })
```

Comment:
This will cause warnings because the state could be changed after the component is unmounted. You should use [Bluebird](http://bluebirdjs.com/) promises and cancel when the component is unmounted. There is already `usePromiseEffect` hook tp handle that.

See: https://reactjs.org/blog/2015/12/16/ismounted-antipattern.html

Reply:
Done, as discussed in Slack.